### PR TITLE
Fix tinymce for production envs

### PR DESCRIPTION
**Pull request by emyl3**
### JIRA issue link
Ensures the tinymce assets are copied to our public assets folder in production envs. Moves all the javascript to the `active_admin.js` file.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/assets/javascripts/active_admin.js [hunk 1]
```diff
@@ -136,6 +124,72 @@ const MAX_DESCRIPTION_LENGTH = 140;
...
+      var componentId = $(e.target).data('componentId');
+      var typeText;
+      if (componentType === 'PageAccordionComponent') {
+        typeText = 'accordion';
+      } else if (componentType === 'PageParagraphComponent') {
+        typeText = 'paragraph';
+      }
+      var componentTextareaId = '#page_page_components_attributes_' + typeText + '_' + componentId + '_component_attributes_text'
+      if (componentType === 'PageAccordionComponent' || componentType === 'PageParagraphComponent') {
+        _initTinyMCE(componentTextareaId)
```
Comment (JoshingYou1) [inline app/assets/javascripts/active_admin.js:158]: nit: just needs a semicolon.

**Review by JoshingYou1 — COMMENTED** [review]
LGTM. Just one small nit. Nice work Elisa 😸

**Review by bradjohnson92008 — APPROVED** [review]
Everything working as it did locally for me.  Thanks Elisa!

**Review by JoshingYou1 — APPROVED** [review]
Great job Elisa! 😸

File: app/assets/javascripts/active_admin.js (hunk 1 above)
Comment (emyl3) [reply app/assets/javascripts/active_admin.js]: ok added!

**Pull request merged**